ydb/core/tx/columnshard/blobs_action/tier/storage.h

```diff
@@ -14,10 +14,10 @@ class TOperator: public IBlobsStorageOperator {
     using TBase = IBlobsStorageOperator;
     const ui64 TabletId;
     const NActors::TActorId TabletActorId;
-    TAtomicCounter CurrentOperatorIdx = 0;
-    std::deque<NWrappers::NExternalStorage::IExternalStorageOperator::TPtr> ExternalStorageOperators;
     std::shared_ptr<TGCInfo> GCInfo = std::make_shared<TGCInfo>();
+
     NWrappers::NExternalStorage::IExternalStorageConfig::TPtr ExternalStorageConfig;
+    TSpinLock ChangeOperatorLock;
```

Comment:
What about using TAdaptiveLock?

Reply:
TAdaptiveLock is overkill for variable copy